.github/mergify.yml

```diff
@@ -0,0 +1,80 @@
+---
+defaults:
+  actions:
+    queue:
+      name: default
+      method: rebase
+      update_method: rebase
+
+
+# each test should be listed separately, do not use regular expressions:
+# https://docs.mergify.io/conditions.html#validating-all-status-check
+# TODO: Use mergify's recently added 'shared configuration support'
+# to dedup some of the check-x=y repetition in the future.
+queue_rules:
+  - name: default
+    conditions:
+      - check-success=check
+      - check-success=build
+      - check-success=podmanbuild
+      - check-success=dockerbuild
+      - check-success=test
+      - check-success=test-kubernetes
+      - check-success=centos-ci/sink-clustered/mini-k8s-1.25
+
+        - check-success=dpulls
+
+
+pull_request_rules:
+  # Clearing approvals after content changes
+  - name: Remove outdated approvals
+    conditions:
+      - base=master
+    actions:
+      dismiss_reviews:
+        approved: true
+        changes_requested: false
+  # Perform automatic merge on conditions
+  - name: Automatic merge on approval
+    conditions:
+      - check-success=check
+      - check-success=build
+      - check-success=podmanbuild
+      - check-success=dockerbuild
+      - check-success=test
+      - check-success=test-kubernetes
+      - check-success=centos-ci/sink-clustered/mini-k8s-1.25
+
```

Comment:
```
- check-success=dpulls
```

is missing here.

Reply:
This is still missing in the latest version.

Reply:
> 

added now

Reply:
strange, I added it  and see it locally, but not in the github view ... 🤷